src/app/forms/components/select/select.component.scss

```diff
@@ -0,0 +1,32 @@
+.govuk-select {
+  min-width: unset;
+  width: 100%;
+}
+
+.govuk-input--width-30 {
+  max-width: 29.5em !important;
+}
+
+.govuk-input--width-20 {
+  max-width: 20.5em !important;
+}
+
+.govuk-input--width-10 {
+  max-width: 11.5em !important;
+}
+
+.govuk-input--width-5 {
```

Comment:
Are these overriding the existing govuk-input--width classes?

Reply:
The `govuk-select` class is setting the `max-width` which is not allowing us to resize it. Here I am appending the `!important` tag to the `govuk-input-width` classes so that the resizing works.

Reply:
That's so bizarre, I remember us getting it to override `govuk-input` in the `text-input` component